housing_django/static/js/app.js

```diff
@@ -0,0 +1,43 @@
+$(document).ready(function() {
+  // Setting CSRF token from cookie.
```

Comment:
Can you move this into some utility js library? Maybe create a js file called util.js and put this in there, e.g.

var Util = {
  getCookie: function(...) {...}
}